dspace-xmlui-mirage2/src/main/webapp/xsl/core/forms.xsl

```diff
@@ -549,7 +571,43 @@
         </div>
     </xsl:template>
 
+    <!-- Since all normalFields can have a language tag, apply it when needed.
+        There may be a better way to do this.
+    -->
     <xsl:template match="dri:field" mode="normalField">
+        <xsl:choose>
+            <xsl:when test="dri:params/dri:param[@name = 'langs']">
+                <div class="row">
+                    <div class="col-xs-9">
+                        <xsl:apply-templates select="." mode="simpleNormalField"/>
+                    </div>
+                    <div class="col-xs-3">
+                        <xsl:variable name="nro" select="count(dri:instance)+1"/>
+                            <xsl:variable name="name">
+                                <xsl:choose>
+                                    <xsl:when test="dri:params[@operations='add delete']">
+                                        <xsl:value-of select="concat(@n, '_', $nro)"/>
+                                    </xsl:when>
+                                    <xsl:otherwise>
+                                        <xsl:value-of select="@n"/>
+                                    </xsl:otherwise>
+                                </xsl:choose>
+                            </xsl:variable>
```

Comment:
This `xsl:variable` block is indented one level too far. (Sorry for the nitpicking, but it looks like it's a child of the variable `nro` above.)